client/my-sites/earn/ads/wrapper.tsx

```diff
@@ -39,45 +36,49 @@ import { isSiteWordadsUnsafe } from 'calypso/state/wordads/status/selectors';
 import './style.scss';
 import 'calypso/my-sites/stats/stats-module/style.scss';
 
-const AdsWrapper = ( { section, children } ) => {
+type AdsWrapperProps = {
+	section?: string;
+	children?: ReactNode;
+};
+
+const AdsWrapper = ( { section, children }: AdsWrapperProps ) => {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
 	const site = useSelector( ( state ) => getSelectedSite( state ) );
-	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
```

Comment:
Several of the selectors used in this file expected siteId to be of type `number | undefined`. Rather than refactoring those or passing 'siteId ?? undefined', I've just updated to site?.ID. All other selectors/method/components can handle site?.ID being undefined except one, and for that I passed `site?.ID ?? null`.